api/api.go

```diff
@@ -27,10 +27,16 @@ import (
 )
 
 const (
-	configAPIListen   = "api_listen"
-	configExpVarPath  = "api_expvar_path"
-	configReadyPath   = "api_ready"
-	configHealthyPath = "api_healthy"
+	configAPIListen         = "api_listen"
+	configExpVarPath        = "api_expvar_path"
+	configReadyPath         = "api_ready"
+	configHealthyPath       = "api_healthy"
+	ConfigDBMaxConns        = "db_config_max_conns"
+	ConfigDBIdleConns       = "db_config_idle_conns"
+	ConfigDBConnsTimeout    = "db_config_conns_timeout_seconds"
+	dbDefaultMaxConnsLimit  = 1000
+	dbDefaultIdleConnsLimit = 1000
+	dbMaxConnTimeoutLimit   = 120
```

Comment:
Could you add these new configs to apid_config_sample.yaml and README.md in 30x/apid?

Reply:
I did want this to be publicised in apid_config_sample.yaml, as this could result in unwanted tampering. This should be altered only under our guidance. I wanted to update Readme.md, but saw we have never updated these kinds of configuration details. Yes, we can update Readme.md.